### ISSUE TITLE
Wrong element clicked for Firefox

### ISSUE THREAD

**Issue by peonone** [Xetera/ghost-cursor#47]
Hi,

I'm using the `click` functionality and it does not work well for Firefox, usually another element below the actual target one gets clicked.
It can be reproduced by this script
```Javascript
const puppeteer = require('puppeteer');
const { createCursor } = require("ghost-cursor");

(async () => {
    const browser = await puppeteer.launch({ product: "firefox", headless: false });
    const page = await browser.newPage();
    const cursor = createCursor(page);
    await page.goto("https://httpbin.org/");
    const selector = "#operations-tag-HTTP_Methods";
    await page.waitForSelector(selector);
    await cursor.click(selector);
})();
```

The intention of this script is to click the "HTTP Methods" section but it was not clicked, instead, another section between "Cookies" to "Anything" got clicked.
There is a debug level logging indicating it was using the JS scroll fallback which seems to be relevant to the issue.
```
Falling back to JS scroll method ProtocolError: Protocol error (DOM.scrollIntoViewIfNeeded): DOM.scrollIntoViewIfNeeded RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
UnknownMethodError@chrome://remote/content/cdp/Error.jsm:112:7
```

Version: 1.1.8

**Comment by Niek** [comment]
I just published v1.1.9 which should fix this - can you please re-test?

**Comment by peonone** [comment]
Thanks @Niek , it works!
